src/MongoDB.Bson/Properties/AssemblyInfo.cs

```diff
@@ -14,21 +14,10 @@
 */
 
 using System;
-using System.Reflection;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-// General Information about an assembly is controlled through the following 
```

Comment:
All these attributes have been removed from all of the AssemblyInfo.cs files because they are automatically generated by MSBuild now.